src/it/JENKINS-45740-metadata/pom.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.jenkins-ci.plugins</groupId>
+        <artifactId>plugin</artifactId>
+        <version>3.29</version>
+    </parent>
+    <groupId>org.jenkins-ci.tools.hpi.its</groupId>
+    <artifactId>JENKINS-45740-metadata</artifactId>
+    <version>1.0-SNAPSHOT</version>
+    <packaging>hpi</packaging>
+    <properties>
+        <java.level>8</java.level>
+        <hpi-plugin.version>@project.version@</hpi-plugin.version>
+        <plugin.info.changelogUrl>http://plugins.jenkins.io/JENKINS-45740-metadata/changelog</plugin.info.changelogUrl>
+        <plugin.info.logoUrl>http://plugins.jenkins.io/JENKINS-45740-metadata/logo.png</plugin.info.logoUrl>
```

Comment:
Are these realistic examples? I would have thought the changelog URL would be `…/releases` and a logo URL would be a GitHub raw content link.

I guess for purposes of the test they could be any well-formed URLs?

Reply:
JCasC, BlueOcean, unofficial Pipeline logos.
Also, many tool/service integration plugins can star using tool logos

Reply:
If the changelog url points to the plugin site, how will the plugin site know where to get changelogs?

Reply:
Like one value would be for update center and one plugins site? Or would update center just always assume plugin site

Reply:
This is just a sample URL, does not really matter what;s the content. In real use-cases I expect it to link to GitHub Releases or other similar sources. Probably it is a good time to start writing documentation for the plugin...